js/Timer_JS_5-6.js

```diff
@@ -0,0 +1,136 @@
+var oTimer = {
+
+  iStartTime: 0,
+  iTimeNow: 0,
+  iPrintTime: 0,
+  iStopTime: 0,
+  iSplitTime: 0,
+  timerId: 0,
+  bActiveTimer: false,
+
+
+  /**
+   * sets timer's time of start
+   */
+  startTimer: function () {
+    iStartTime = Date.now() - this.iPrintTime;
+  },
+
+  /**
+   * sets time since timer was started
+   * @returns {number} shows time since timer was started
+   */
+  getTime: function () {
+    var iTimeNow = Date.now();
+    return iTimeNow - iStartTime;
+  },
+
+  /**
+   * prints time every 1 msec since timer was started
+   */
+  printTime: function () {
+    this.startTimer();
+    this.getTime();
+    timerId = setInterval(function () {
+      var mlsec = oTimer.getTime();
+      var sec = new Date(mlsec);
+      document.querySelector('.timestring').innerHTML = oTimer.checkTime((sec.getHours() - 2)) + ':' + oTimer.checkTime(sec.getMinutes()) + ':' + oTimer.checkTime(sec.getSeconds()) + '.' + oTimer.checkTimeMsec(sec.getMilliseconds());
+    }, 1);
+    this.bActiveTimer = true;
+    document.querySelector('.startbtn').style.display = 'none';
+    document.querySelector('.pausebtn').style.display = 'inline-block';
+  },
+
+  /**
+   * pauses timer and prints to the new string below the time since timer's start till pause
+   */
+  pauseTimer: function () {
+    clearInterval(timerId);
+    this.iPrintTime = this.getTime();
```

Comment:
All of the 4 calls of `getTime()` can return you different result. If you need same moment in all cases call the method once and store the result to local var

Reply:
fixed - other 3 calls were replaced with this.iPrintTime